이거.js

```diff
@@ -0,0 +1,119 @@
+let today = new Date();
+
+//오늘의 현재 요일 표기
+let dayweek = today.getDay();
+let week = new Array(
+    "SUN",
+    'MON',
+    "TUE",
+    "WED",
+    "THU",
+    "FRI",
+    "SAT");
+let day_of_week = week[dayweek]
+
+//오늘의 현재 날짜 표기
+let dd = today.getDate();
+
+//오늘의 현재 월 표기
+let mm = today.getMonth();+1
+let month = new Array(
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+    "Oct",
+    "Nov",
+    "Dec");
+let year_of_month = month[mm]
+
+//오늘의 현재 연도 표기
+let yyyy = today.getFullYear();
+
+//캘린더 요일과 일 표기
+document.getElementById('current-day').innerText = day_of_week;
+document.getElementById('current-date').innerText = dd;
+
+//캘린더 월/년 표기 ---------이 두개 나란하게 표시 어떻게 해야하나요?
+document.getElementById('current-mon').innerText = year_of_month;
+document.getElementById('current-yr').innerText = yyyy;
+
+
+
+//달력만들기
+//top = th 요일 tr = 1,2,3,4,5,week , td = day
+function createCalendar(elem, year, month) {
+
+    let mon = month - 1; // months in JS are 0..11, not 1..12
+    let d = new Date(year, mon);
+
+    let table = 
+    '<table><tr><th>MO</th><th>TU</th><th>WE</th><th>TH</th><th>FR</th><th>SA</th><th>SU</th></tr><tr>';
+
+
+    // spaces for the first row
+    // from Monday till the first day of the month
+    // * * * 1  2  3  4
+    for (let i = 0; i < getDay(d); i++) {
+      table += '<td></td>';
+    }
+
+    // <td> //현재 월의 마지막 날짜까지 달력에 표기하기
+    while (d.getMonth() == mon) {
```

Comment:
동등 비교 연산자 (`==`)는 좌항과 우항의 타입이 다르더라도 암묵적 타입 변환을 통해 타입을 일치시킨 후 같은 값을 갖는지 비교합니다. 따라서 동등 비교 연산자는 많은 부작용을 일으키므로 일치 비교 연산자(`===`)를 사용하시는 걸 권장드립니다.

```suggestion
    while (d.getMonth() === mon) {
```

- [poiemaweb.com/js-operator#6-비교-연산자](https://poiemaweb.com/js-operator)